src/calculateRentalCost.js

```diff
@@ -4,7 +4,22 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const carCostPerDay = 40;
+  const mediumRentTermDiscount = 20;
+  const longRentTermDiscount = 50;
+  const price = carCostPerDay * days;
+
+  if (days < 3) {
+    return price;
+  }
+
+  if (days >= 3 && days <= 6) {
```

Comment:
Instead of using 'days >= 3 && days <= 6', you can simplify the condition to 'days < 7' since the previous return statement already handles the case when 'days < 3'.